src/zcash/Proof.cpp

```diff
@@ -202,6 +202,7 @@ r1cs_ppzksnark_proof<curve_pp> ZCProof::to_libsnark_proof() const
 
 ZCProof ZCProof::random_invalid()
 {
+    initialize_curve_params();
```

Comment:
This function no longer exists, and the declaration of it in src/zcash/Proof.hpp should be removed.

Reply:
Removed in merge commit.